### ISSUE TITLE
Configuration in Readme.md doesn't work 100%

### ISSUE THREAD

**Issue by craiglondon** [assayer-pro/yii2-cspreport#2]
I had to change this line

```
            'parsers' => [
                'application/json' => 'yii\web\JsonParser',
            ],

```
to

```
            'parsers' => [
                'application/csp-report' => 'yii\web\JsonParser',
            ],
```

I tested this to work with FireFox and Chrome.

**Comment by serge-larin** [comment]
You're right! But this config worked with yii version 2.0.10 and don't working with 2.0.15.
I fixed it in commit b4cc31b.

**Comment by craiglondon** [comment]
Thanks for the quick turn around
